src/ripple/proto/ripple.proto

```diff
@@ -8,6 +8,8 @@ enum MessageType
     mtPING                  = 3;
     mtPROOFOFWORK           = 4;
     mtCLUSTER               = 5;
+    mtGET_SHARD_INFO        = 10;
+    mtSHARD_INFO            = 11;
```

Comment:
These messages were previously used for peerfinder messages; while those messages are obsolete (and I doubt that any server out there is still sending or receiving them) I think we should avoid reusing message identifiers.